diff -r 26bbec7257ab test/gtest/include/gtest/gtest-typed-test.h
--- a/test/gtest/include/gtest/gtest-typed-test.h	Tue Feb 19 22:56:58 2019 +0000
+++ b/test/gtest/include/gtest/gtest-typed-test.h	Tue Feb 19 16:26:40 2019 -0800
@@ -31,7 +31,9 @@
 
 #ifndef GTEST_INCLUDE_GTEST_GTEST_TYPED_TEST_H_
 #define GTEST_INCLUDE_GTEST_GTEST_TYPED_TEST_H_
+#ifndef _WIN32
 #pragma GCC system_header
+#endif
 
 // This header implements typed tests and type-parameterized tests.
 
diff -r 26bbec7257ab test/gtest/src/gtest.cc
--- a/test/gtest/src/gtest.cc	Tue Feb 19 22:56:58 2019 +0000
+++ b/test/gtest/src/gtest.cc	Tue Feb 19 16:26:40 2019 -0800
@@ -33,7 +33,9 @@
 
 #include "gtest/gtest.h"
 #include "gtest/gtest-spi.h"
+#ifndef _WIN32
 #pragma GCC system_header
+#endif
 
 #include <ctype.h>
 #include <math.h>
diff -r 26bbec7257ab test/gtest/include/gtest/internal/gtest-port.h
--- a/test/gtest/include/gtest/internal/gtest-port.h	Tue Feb 19 22:56:58 2019 +0000
+++ b/test/gtest/include/gtest/internal/gtest-port.h	Tue Feb 19 16:26:40 2019 -0800
@@ -39,7 +39,9 @@
 
 #ifndef GTEST_INCLUDE_GTEST_INTERNAL_GTEST_PORT_H_
 #define GTEST_INCLUDE_GTEST_INTERNAL_GTEST_PORT_H_
+#ifndef _WIN32
 #pragma GCC system_header
+#endif
 
 // The user can define the following macros in the build script to
 // control Google Test's behavior.  If the user doesn't define a macro
diff -r 26bbec7257ab test/gtest/include/gtest/gtest-printers.h
--- a/test/gtest/include/gtest/gtest-printers.h	Tue Feb 19 22:56:58 2019 +0000
+++ b/test/gtest/include/gtest/gtest-printers.h	Tue Feb 19 16:26:40 2019 -0800
@@ -94,7 +94,9 @@
 
 #ifndef GTEST_INCLUDE_GTEST_GTEST_PRINTERS_H_
 #define GTEST_INCLUDE_GTEST_GTEST_PRINTERS_H_
+#ifndef _WIN32
 #pragma GCC system_header
+#endif
 
 #include <ostream>  // NOLINT
 #include <sstream>
